faq/debugging.inc

```diff
@@ -239,7 +239,12 @@ shell$ mpirun -np 4 xterm -e gdb my_mpi_application
 </geshi>
 
 If running on a personal computer, this will probably work.
-Unfortunately, it likely _won't_ work on an computing cluster.
+You can also use " . aurchin_click("https://github.com/Azrael3000/tmpi") .
+"<code>tmpi</code></a> to launch the debuggers in separate [tmux]
+panes instead of separate [xterm] windows, which has the advantage of
+synchronizing keyboard input between all debugger instances.
+
+Unfortunately, this approach likely _won't_ work on an computing cluster.
```

Comment:
Are you saying that both the `tmpi` _and_ the `xterm` approach won't work on a computing cluster?  If so, it might be a little more clear to state something like:

> Unfortunately, the `tmpi` and `xterm` approaches likely _won't_ work on a computing cluster.

Reply:
Yes, it makes sense that they would have the same limitation.  I'll tweak the wording. Thanks!